src/lib/utils/monitoring.ts

```
@@ -0,0 +1,36 @@
+// Simple error tracking utility
+interface ErrorData {
+  message: string;
+  stack?: string;
+  url: string;
+  timestamp: number;
+  userAgent: string;
+  metadata?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)